x/data/server/keys.go

```diff
@@ -1,50 +1,56 @@
 package server
 
-import (
-	"encoding/base64"
-)
+import sdk "github.com/cosmos/cosmos-sdk/types"
 
 const (
-	AnchorTablePrefix byte = 0x0
-	CIDSignerPrefix   byte = 0x1
-	SignerCIDPrefix   byte = 0x2
-	DataTablePrefix   byte = 0x3
+	IriIDTablePrefix      byte = 0x0
+	AnchorTimestampPrefix byte = 0x1
+	IDSignerPrefix        byte = 0x2
+	SignerIDPrefix        byte = 0x2
+	RawDataPrefix         byte = 0x3
```

Comment:
Shouldn't `IDSignerPrefix` and `SignerIDPrefix` be distinct ?
```suggestion
	IDSignerPrefix        byte = 0x2
	SignerIDPrefix        byte = 0x3
	RawDataPrefix         byte = 0x4
```